themes/gatsby-theme-blog-mdx/src/tokens/typography.js

```diff
@@ -0,0 +1 @@
+export { default } from "typography-theme-bootstrap"
```

Comment:
long-term we're going to have to figure out a solution for the "global" typography styles I think.